docs/pages/zero-trust-access/management/guides/awsoidc-integration-console.mdx

```diff
@@ -0,0 +1,200 @@
+---
+title: AWS CLI/Console Access Wizard
+description: Access AWS CLI and Console using AWS OIDC integration.
+tags:
+ - how-to
+ - zero-trust
+ - infrastructure-identity
+ - aws
+---
+
+The AWS OIDC integration allows Teleport users to access AWS CLI and Console using AWS IAM Roles.
+
+You can set up access using the "AWS CLI/Console Access via AWS OIDC IdP" enrollment wizard from the UI or you can follow this guide to set it up manually.
+
+## How it works
+
+### AWS OIDC Integration
+
+Setting up access to AWS CLI and Console requires an [AWS OIDC integration](./awsoidc-integration.mdx).
+The AWS OIDC integration creates and configures an AWS IAM OpenID Connect Identity Provider (OIDC IdP) and an AWS IAM role that your Teleport cluster can assume.
+
+The enrollment wizard adds permissions to the AWS OIDC integration IAM role to assume other existing IAM Roles.
+
+## Prerequisites
+
+- A running Teleport cluster
+- An AWS account and permissions to create IAM Identity Providers and roles
+
+## Step 1/2. Configure RBAC
+
+Whether you use the enrollment wizard or set up access manually, you need access to the `editor` role.
+
+## Step 2/2. Configure access using the wizard
+
+The enrollment wizard is available in the "Add New Resource" panel of the Teleport Web UI:
+
+![Screenshot of AWS CLI/Console Access via AWS OIDC IdP enrollment wizard tile](../../../../img/aws/enroll-aws-access-awsoidc.png)
+
+The Teleport Web UI walks you through the steps to set up an AWS OIDC integration (if you don't have one already), configure IAM permissions and set up access to AWS IAM Roles.
+
+Set the <Var name="integration-iam-role"/> field to the IAM Role name created by the AWS OIDC integration.
+
+### Allow Integration to assume other IAM Roles
+
+The wizard instructs you to run a script, which will configure the necessary permissions for the AWS OIDC integration IAM role to assume other IAM Roles:
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": "sts:AssumeRole",
+            "Resource": "*",
+            "Condition": {
+                "StringEquals": {
+                    "iam:ResourceTag/teleport.dev/integration": "true"
+                }
+            }
+        }
+    ]
+}
+```
+
+### Requirements for other IAM Roles to be accessible from Teleport
+
```

Comment:
To make things easier for users, is there a quick `aws` CLI command that will add the tag and Trust Relationship?